get-started/configuration.md

```diff
@@ -6,13 +6,26 @@ publish: true
 position: 30
 ---
 
-## Configuration
+## HTTPS Configuration
 
-The Fiddler Everywhere client needs an account with administrative rights to enable secure traffic capturing (HTTPS). Without installing trust root certificate via admin account, you will be able to capture only non-secure HTTP traffic.
+By default, Fiddler Everywhere client intercepts insecure traffic (**HTTP**) only and needs an account with administrative rights to capture secure traffic (**HTTPS**). The Fiddler Everywhere client act as a man-in-the-middle (against the HTTPS traffic). To achieve that, the proxy must generate a root certificate and use that root certificate to create multiple end-entity certificates, one for each intercepted HTTPS site. 
+
+Generate and trust the root certificate via the options provided in [**_Settings > HTTPS_**]({%slug decrypt-https-traffic%}) section. For specific related to enabling HTTPS capturing on the different operating system see the links below:
```

Comment:
1) For specific -> specific**s**. Consider rephrasing to "For more details on how to enable HTTPS capturing..."
2) Comma between "system" and "see"